apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsImageControl.tsx

```diff
@@ -51,6 +55,36 @@ export function JsonFormsImageControl({
     }
   }, [modifiedAssets, path, pendingAsset, setModifiedAssets])
 
+  useEffect(() => {
+    const urlToFile = async (
+      url: string,
+      filename: string,
+      mimeType: string,
+    ) => {
+      const response = await fetch(url)
+      const blob = await response.blob()
+      return new File([blob], filename, { type: mimeType })
+    }
+
+    async function convertImage(url: string) {
+      const fileName = url.split("/").pop()
+      const fileType = `image/${url.split(".").pop()}`
+      const imageUrl = url.startsWith("/")
+        ? `${NEXT_PUBLIC_S3_PUBLIC_ASSETS_DOMAIN_NAME}${url}`
+        : url
+      const file = await urlToFile(imageUrl, fileName || "", fileType)
+      setPendingAsset({ path, src: imageUrl, file })
+    }
+
+    if (!data) {
+      return
+    }
+
+    convertImage(data)
```

Comment:
lint

Reply:
Disabled the eslint rule in d17406f2c80ade0533a78f923ca9d9c970556c61.